x/crosschain/keeper/v2_zevm_inbound.go

```diff
@@ -137,6 +173,103 @@ func (k Keeper) ProcessZEVMInboundV2(
 
 		EmitZRCWithdrawCreated(ctx, *cctx)
 	}
-
 	return nil
 }
+
+// getZetaInboundDetails retrieves the details for a ZETA withdrawal event, it returns an InboundDetails object
+func (k Keeper) getZetaInboundDetails(
```

Comment:
```suggestion
func (k Keeper) getZETAInboundDetails(
```
Need to use uppercase format when mentioning the token